roles/ansible-test-splitter/files/split_targets.py

```diff
@@ -38,13 +38,13 @@
 
 random.shuffle(regular_targets)
```

Comment:
Given the use of `[x::remaining_jobs]` I'd recommend removing the shuffle here.  We already know that in general slower tests tend to use similar prefixes